sale_cancel_reason/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2013 Guewen Baconnier, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Sale Cancel Reason',
+    'version': '11.0.1.0.0',
+    'author': "Camptocamp,Odoo Community Association (OCA)",
+    'category': 'Sale',
+    'license': 'AGPL-3',
+    'website': "http://www.camptocamp.com",
```

Comment:
Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)